docs/proposals/kubelet-eviction.md

```diff
@@ -69,6 +69,7 @@ The `kubelet` will support the ability to trigger eviction decisions on the foll
 | Eviction Signal  | Description                                                                     |
 |------------------|---------------------------------------------------------------------------------|
 | memory.available | memory.available := node.status.capacity[memory] - node.stats.memory.workingSet |
+| disk.available   | disk.available := node.stats.fs.available |
```

Comment:
@vishh - now with a more detailed scenario that demonstrates why image file system must have a separate threshold than root file system usage.  Literally all of our production customers will run them separate, and the image file system will be much larger.

Typical production example:

**imagefs**
* 200 GB disk
* `--dm.basesize=3GB` to restrict container
* alert at 10% free, 5% free when they can grow dynamically on cloud
** bare metal will want to have a greater reserve most-likely

**rootfs**
* 16 GB disk on /var
* alert at 10% and 5% free
* treat empty dir like /tmp so assume 1 GB scratch space

Many operations teams actually want to pro-actively clean images to the used set, but at a scheduled interval to control for resource usage rather than in the face of disk pressure.  So I got some feedback that at the soft-threshold type scenario, some operators would ignore them in favor of running more scheduled maintenance to prune the imagefs, but hard thresholds are of course of value.

Keep in mind in this environment, usage of empty dir was heavily restricted in favor of directing users towards persistent volumes, not sure all users will treat rootfs uniformly.

Either way, with disk sizes so different, having a uniform threshold was deemed undesirable.